janusgraph-codepipelines-ci/pom.xml

```diff
@@ -65,7 +65,7 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-log4j12</artifactId>
             <version>1.7.12</version>
-            <scope>runtime</scope>
+            <scope>optional</scope>
```

Comment:
Is `<scope>optional</scope>` valid? Was this meant to be `<optional>true</optional>`?

According to [Maven docs](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope), the valid scopes are: `compile`, `provided`, `runtime`, `test`, `system`, and `import`.

Reply:
@mbrukman Yep good catch. I'll push the fix later today.